source/form.html

```diff
@@ -7,6 +7,9 @@
     <link rel="icon" type="img/svg" sizes="32x32" href="./favicon32.svg">
     <link rel="apple-touch-icon" type="img/png" sizes="152x152" href="./img/favicons/favicon152.png">
     <link rel="stylesheet" href="./css/style.min.css">
+    <link rel="preload" href="/fonts/opensans.woff2" as="font">
+    <link rel="preload" href="/fonts/opensansbold.woff2" as="font">
+    <link rel="preload" href="/fonts/opensanslight.woff2" as="font">
```

Comment:
Чтобы предзагрузка работала, надо добавить еще такие параметры:
```suggestion
    <link rel="preload" href="/fonts/opensans.woff2" as="font" type="font/woff2" crossorigin>
    <link rel="preload" href="/fonts/opensansbold.woff2" as="font" type="font/woff2" crossorigin>
    <link rel="preload" href="/fonts/opensanslight.woff2" as="font" type="font/woff2" crossorigin>
```

Без них браузер по каким-то там причинам не разбирает что это тот же файл и все равно в стилях грузит еще раз его после прелоада.